helpers/python.ts

```diff
@@ -155,9 +155,17 @@ const getAdditionalDependencies = (
       });
       break;
     case "openai":
+      dependencies.push({
+        name: "llama-index-llms-openai",
```

Comment:
@marcusschiesser these openai provider packages are required when upgrading llama-index v0.11 but multiagent template is having issue with these package right now.

```
Because no versions of llama-agents match >0.0.14,<0.0.15
 and llama-agents (0.0.14) depends on llama-index-core (>=0.10.50,<0.11.0), llama-agents (>=0.0.14,<0.0.15) requires llama-index-core (>=0.10.50,<0.11.0).
And because llama-index-llms-openai (0.2.0) depends on llama-index-core (>=0.11.0,<0.12.0)
 and no versions of llama-index-llms-openai match >0.2.0,<0.3.0, llama-agents (>=0.0.14,<0.0.15) is incompatible with llama-index-llms-openai (>=0.2.0,<0.3.0).
So, because app depends on both llama-index-llms-openai (^0.2.0) and llama-agents (^0.0.14), version solving failed.
```

So, we may have two options:
+ Add a condition here to use different version for multiagent template.
+ Temporarily accept the issue and it should be fixed once the newer `llama-agent` package support those OpenAI provider dependencies.